### PR TITLE
Generic architecture

### PR DESCRIPTION
**Pull request by soniktrooth**
First stab at #20. Got the home page and team pages happening. Thoughts?

### CODE REVIEW

File: src/index.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
 ---
 template: home.html
+title: Who needs a CMS?
 ---
 
-## KalaStatic
-
-This is the homepage.
-
+Not me, that's for sure. CMS's are for whimps and loosers so suck it up and generate your static site now!
```
Comment (andrewmallis) [inline src/index.md]: :+1:

**Comment by andrewmallis** [comment]
Generally: great work. 
Good humour. Solid.

There is more in the this pull request than I'd like to see.

It would be ok to have the contact page here, but the form should be committed to #5

Search is not part of the generic architecture. 
Commits for this should be against #24

The complications in these areas holds up the merging.

**Comment by andrewmallis** [comment]
also homepage implementation is debatable. Agree that a hero + 3 col layout is "generic" but I think we want our static site start states to be exceptional.

I have opened #30 to work this out. Perhaps commit against that?

**Comment by soniktrooth** [comment]
Ok, I've pulled out all the objectionable stuff into separate pull requests referencing their various issues.

**Comment by madeofpeople** [comment]
A little bit o manual merging, pulling it in.

**Pull request merged**